src/main/java/frc/robot/commands/ClimbKeepDown.java

```diff
@@ -5,6 +5,7 @@
 package frc.robot.commands;
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.Constants;
 import frc.robot.subsystems.ClimbSubsystem;
 
 public class ClimbKeepDown extends CommandBase {
```

Comment:
ClimbKeepDown seems like it needs a new name since it does more than keep it down

Reply:
I have renamed it to Climb Periodic bc every thing in it need to run periodicaly